en/products/relay/features.ftl

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/relay/
+
+## HIGHLIGHTED FEATURES SECTION
+
+highlighted-features-section-title = Secure, simple features to help protect your identity
+highlighted-features-section-bottom-title = Protect your identity (and your inbox) with { -brand-name-firefox-relay }
+highlighted-features-section-bottom-cta = Get started
+
+highlighted-features-section-unlimited-masks-headline = Create unlimited email masks
+# Variables:
+#   $mask_limit (number) - the number of masks included with a particular plan
+highlighted-features-section-unlimited-masks-body = Everyone gets { $mask_limit } email masks for free.
+    But with { -brand-name-relay-premium }, you can generate as many masks as you need to help protect your email inbox
+    from spammers, hackers, and online trackers.
```

Comment:
```suggestion
highlighted-features-section-unlimited-masks-body-v2 = {$mask_limit ->
  [one] Everyone gets { $mask_limit } email mask for free. But with { -brand-name-relay-premium }, you can generate as many masks as you need to help protect your email inbox from spammers, hackers, and online trackers.
  *[other] Everyone gets { $mask_limit } email masks for free. But with { -brand-name-relay-premium }, you can generate as many masks as you need to help protect your email inbox from spammers, hackers, and online trackers.
```

This should have plural. Even if the English version will only be plural, we should trigger plurals for other more complex plural locales.